plugins/ecs-bridge/commands/commands.go

```diff
@@ -139,13 +139,12 @@ func del(args *skel.CmdArgs, engine engine.Engine) error {
 
 	if utils.ZeroOrNil(conf.IPAM) {
 		detailLog("IPAM configuration not found, skip DEL for IPAM", args, conf, "")
-		return nil
-	}
-
-	detailLog("Running IPAM plugin DEL", args, conf, "")
-	err = engine.RunIPAMPluginDel(conf.IPAM.Type, args.StdinData)
-	if err != nil {
-		return err
+	} else {
+		detailLog("Running IPAM plugin DEL", args, conf, "")
+		err = engine.RunIPAMPluginDel(conf.IPAM.Type, args.StdinData)
+		if err != nil {
+			return err
```

Comment:
Do we want to log a warning here and continue on with veth deletion? I believe IPAM resource management is done separately from the actual veth management, and even if one of them fails we can should still attempt the other.